From ff37b165bdb100450c7996c9fac0fad2e6ffe31d Mon Sep 17 00:00:00 2001
From: Ying Zhang <ying.zhang22455@nxp.com>
Date: Thu, 29 Sep 2016 11:40:37 +0800
Subject: [PATCH 121/124] armv8: aarch32: defconfig: Enable support for AHCI
 SATA

This patch is to enable support for the Freescale QorIQ AHCI SoC's
onboard AHCI SATA.

Signed-off-by: Alison Wang <alison.wang@nxp.com>
---
 arch/arm/configs/ls_aarch32_defconfig |    5 +++++
 1 file changed, 5 insertions(+)

--- a/arch/arm/configs/ls_aarch32_defconfig
+++ b/arch/arm/configs/ls_aarch32_defconfig
@@ -92,7 +92,12 @@ CONFIG_MTD_SPI_NOR=y
 CONFIG_MTD_SPI_NOR_USE_4K_SECTORS=y
 CONFIG_SPI_FSL_QUADSPI=y
 CONFIG_BLK_DEV_SD=y
+CONFIG_CHR_DEV_SG=y
 CONFIG_ATA=y
+CONFIG_SATA_AHCI=y
+CONFIG_SATA_AHCI_PLATFORM=y
+CONFIG_AHCI_QORIQ=y
+CONFIG_SATA_SIL24=y
 CONFIG_BLK_DEV_LOOP=y
 CONFIG_BLK_DEV_RAM=y
 CONFIG_BLK_DEV_RAM_COUNT=8
